Makefile

```diff
@@ -20,7 +20,7 @@ test: generate fmt vet manifests
 	rm -f cover.out cover.out.tmp cover.json
 
 # Run tests with existing cluster
-test-existing: generate fmt vet manifests
+test-existing: generate fmt checks manifests
```

Comment:
Is it possible to have ` generate fmt vet manifests lint` instead of ` generate fmt checks manifests`?

Reply:
As mentioned above, golangci-lint includes go vet. Let me know if you still want go vet added back

Reply:
Re-added as per discussion